.github/workflows/platform-project-automation.yml

```diff
@@ -21,14 +22,44 @@ jobs:
       - name: Set current date env var
         id: set_date
         run: echo ::set-output name=CURRENT_DATE::$(date +'%Y-%m-%dT%H:%M:%S%z')
-      
-      - name: Add issue to project if labelled with area/platform
+
+      - name: Add issue to the cloud project if labelled with area/platform and not area/platform-oss
```

Comment:
This is a good idea, but I wonder what happens if somebody first adds the `area/platform` label and then immediately adds the `area/platform-oss` label? I guess we can just see what happens in practice and adjust if needed

Reply:
that's a good point. I think that for now we can make sure that the board have an appropriated filter